iocage_lib/ioc_plugin.py

```diff
@@ -84,6 +87,137 @@ def __init__(self, release=None, jail=None, plugin=None, branch=None,
             # Backwards compat
             self.branch = 'master'
 
+    def clone_repo(self):
+        if self.server == "download.freebsd.org":
+            git_server = "https://github.com/freenas/iocage-ix-plugins.git"
+        else:
+            git_server = self.server
+
+        git_working_dir = f"{self.iocroot}/.plugin_index"
```

Comment:
Wont this require iocage to be activated to show a list of plugins?

Reply:
That is the whole idea as @skarekrow suggested. I have separated the functions, once we have the plugin index and plugin json files data, that data can be passed on to the new function which retrieves versions.